frontends/web/src/components/icon/logo.tsx

```diff
@@ -67,7 +66,7 @@ export const AppLogoInverted = (props: GenericProps) => <img {...props} draggabl
 export const BitBoxSwiss = (props: GenericProps) => <img {...props} draggable={false} src={BitBoxSwissLogo} alt="BitBox" className={style.logo} />;
 export const BitBoxSwissInverted = (props: GenericProps) => <img {...props} draggable={false} src={BitBoxSwissInvertedLogo} alt="BitBox" className={style.logo} />;
 export const Shift = (props: GenericProps) => <img {...props} draggable={false} src={ShiftLogo} alt="Shift Crypto" className={style.logo} />;
-export const SwissMadeOpenSource = (props: GenericProps) => <img {...props} draggable={false} src={SwissOpenSource} alt="Swiss Made Open Source" className={[style.swissOpenSource, props.large ? style.large : ''].join(' ')} />;
+export const SwissMadeOpenSource = ({large: boolean, ...props}: GenericProps) => <img {...props} draggable={false} src={SwissOpenSource} alt="Swiss Made Open Source" className={[style.swissOpenSource, props.large ? style.large : ''].join(' ')} />;
```

Comment:
@BrianCraig `large` is deconstructed but then there is `props.large` in `className={[style.swissOpenSource, props.large ? style.large : '']`

Reply:
This is a problem that `large` is a `<img>` prop, but interferes (not a boolean), and it was passed down to the img tag, when it's a component prop